### ISSUE TITLE
Order of columns in `--pretty` does not always match the order in `SELECT`

### ISSUE THREAD

**Issue by liff** [multiprocessio/dsq#38]
Steps to reproduce:

```shell
$ cat <<EOF > test.csv
first,second,third
1,a,X
2,b,Y
EOF

$ dsq --pretty test.csv 'select first, second, third from {}'
+-------+--------+-------+
| first | second | third |
+-------+--------+-------+
|     1 | a      | X     |
|     2 | b      | Y     |
+-------+--------+-------+

# run the command again and again until you see something like:

$ dsq --pretty test.csv 'select first, second, third from {}'
+--------+-------+-------+
| second | third | first |
+--------+-------+-------+
| a      | X     |     1 |
| b      | Y     |     2 |
+--------+-------+-------+
```

Happens on Linux and M1 Mac.

**Comment by eatonphil** [comment]
Hey! That's a good point. Right now there is no relation at all between your column order and the output. The best I can do in the short-term is alphabetize the columns so at least they are predictable. I'll add support for that.

**Comment by eatonphil** [comment]
This fixed sorting is now in 0.10.0
